src/components/structures/ContextMenu.js

```diff
@@ -421,6 +423,54 @@ MenuItemCheckbox.propTypes = {
     onClick: PropTypes.func.isRequired,
 };
 
+// Semantic component for representing a styled role=menuitemcheckbox
+export const StyledMenuItemCheckbox = ({children, label, onChange, onClose, checked, disabled=false, ...props}) => {
```

Comment:
as a distant future problem: we should consider not allowing multiple components in the same file, as this file is illegible.